### PR TITLE
Demo 6/2022 summary

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of Public Demo 6/2022

### CODE REVIEW

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's sixth session of this year edition of our public demo was held and [streamed live](https://www.youtube.com/watch?v=R6fwb7FiiCI) on September 7th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=R6fwb7FiiCI) Beginning of the session, the introduction
+- [02:02](https://youtu.be/R6fwb7FiiCI?t=122) Project news: PrestaShop 1.7.8.7 release
```
Comment (matks) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md:16]: Do you create the timestamps manually like I did? 😝  maybe you want to script it

**Review by matks — APPROVED** [review]
OK for publication for me @kpodemski

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md (hunk 1 above)
Comment (kpodemski) [reply news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md:16]: yeah, fortunately, it's quite fast. A huge time-saver is that if you paste it to the YT description, it creates proper segments in the video automatically :D So a few minutes here is not a problem :D

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - September 2022 session"
+subtitle: "We held the sixth session of this year open source project's public demo"
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
subtitle: "The open source project's public demo was held for the sixth time this year"
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+date:   2022-09-08 11:00
+authors: [ PrestaShop ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's sixth session of this year edition of our public demo was held and [streamed live](https://www.youtube.com/watch?v=R6fwb7FiiCI) on September 7th, 2022.
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
The project's public demo was held for the sixth time this year and [streamed live](https://www.youtube.com/watch?v=R6fwb7FiiCI) on September 7th, 2022.
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=R6fwb7FiiCI) Beginning of the session, the introduction
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
- [00:00](https://www.youtube.com/watch?v=R6fwb7FiiCI) Beginning of the session: the introduction
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- [02:02](https://youtu.be/R6fwb7FiiCI?t=122) Project news: PrestaShop 1.7.8.7 release
+- [03:26](https://youtu.be/R6fwb7FiiCI?t=206) Project news: Product Comments v5.0.2 release
+- [03:59](https://youtu.be/R6fwb7FiiCI?t=239) Project news: Backward compatibility promise ADR
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
- [03:59](https://youtu.be/R6fwb7FiiCI?t=239) Project news: backward compatibility promise ADR
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- [05:53](https://youtu.be/R6fwb7FiiCI?t=353) Project news: PrestaShop 8.0 Beta
+- [08:31](https://youtu.be/R6fwb7FiiCI?t=511) Demo: How could we implement the PrestaShop hooks system in Vue components? [@NeOMakinG](https://github.com/NeOMakinG)
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
- [08:31](https://youtu.be/R6fwb7FiiCI?t=511) Demo: how could we implement the PrestaShop hooks system in Vue components? [@NeOMakinG](https://github.com/NeOMakinG)
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- [19:19](https://youtu.be/R6fwb7FiiCI?t=1159) Demo: What's new for developers in PrestaShop 8.0? [@kpodemski](https://github.com/kpodemski)
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
- [19:19](https://youtu.be/R6fwb7FiiCI?t=1159) Demo: what's new for developers in PrestaShop 8.0? [@kpodemski](https://github.com/kpodemski)
```

File: news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - September 2022 session"
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md]:
```suggestion
title:  "PrestaShop Project Public Demo - September 2022 Session"
```

**Review by coffrep — DISMISSED** [review]

**Comment by coffrep** [comment]
Hi Krystian, great post and great public demo, I've made a few suggestions on the document.

**Comment by kpodemski** [comment]
Thank you for the reviews @matks @coffrep ! :)

**Pull request merged**